locales/de_DE.json

```diff
@@ -1272,34 +1272,34 @@
     },
     "discourse-sso": {
       "login": {
-        "title": "Melden Sie sich bei der Nash-Gemeinschaft an",
-        "description": "Sie benötigen ein Nash-Konto, um sich bei der Nash-Community anzumelden.",
+        "title": "Melde dich bei der Nash-Community an",
+        "description": "Du benötigst ein Nash-Konto, um dich bei der Nash-Community anzumelden.",
         "button-login": "Anmelden mit Nash-Konto",
         "link-create": "<0>Ein Nash-Konto anlegen</0>"
       },
       "create-account": {
-        "title": "Erstellen Sie Ihr Nash Community-Konto",
-        "description": "Um Nachrichten in der Community zu veröffentlichen, müssen Sie mindestens 1 NEX für 24 Monate einsetzen.",
+        "title": "Erstelle ein Nash Community-Konto",
+        "description": "Um Nachrichten in der Community zu veröffentlichen, müssen du mindestens 1 NEX für 24 Monate staken.",
         "username-label": "Community-Benutzername",
-        "username-placeholder": "Wählen Sie einen Namen...",
-        "button-create": "Gemeinschaftskonto erstellen",
-        "migrate-description": "Wenn Sie die Nash Extension verwenden, um sich bei der Community anzumelden, können Sie Ihr Konto <0>migrieren</0>.",
-        "username-exits": "‘{{username}}’ ist bereits vergeben. Bitte wählen Sie einen anderen Benutzernamen."
+        "username-placeholder": "Wähle einen Namen ...",
+        "button-create": "Community-Konto erstellen",
+        "migrate-description": "Wenn du die Nash Extension verwendest, um sich in dem Community-Forum anzumelden, kannst du dein Konto <0>migrieren</0>.",
+        "username-exits": "‘{{username}}’ ist bereits vergeben. Bitte wähle einen anderen Benutzernamen."
       },
       "migrate-account": {
-        "title": "Migrieren Sie Ihr Nash Community-Konto",
-        "migration-step1": "Melden Sie sich mit der Nash-Erweiterung an.",
-        "migration-step2": "Ändern Sie Ihre Community-E-Mail-Adresse in die von Ihrem Nash-Konto verwendete Adresse.",
-        "migration-step3": "Jetzt reicht's! Sobald Ihre E-Mail-Adresse geändert wurde, können Sie sich über Ihr Nash-Konto anmelden.",
-        "button-migrate": "Anmelden mit der Nash-Erweiterung",
+        "title": "Migriere dein Nash Community-Konto",
+        "migration-step1": "Melde dich mit der Nash-Extension an.",
+        "migration-step2": "Ändere deine Community-E-Mail-Adresse in die von deinem Nash-Konto verwendete Adresse.",
```

Comment:
Kann man vereinfachen? `die von deinem Nash-Konto verwendete Adresse` -- zu kompliziert?

Reply:
Ich wüsste nicht wie :/

Reply:
Relativsatz oder so. Habe geändert: `"Ändere deine Community-E-Mail-Adresse in die Adresse, die dein Nash-Konto verwendet`